nits from Lars (#384)
quicwg/ops-drafts: draft-ietf-quic-applicability.md

## Stream versus Flow Multiplexing

Streams are meaningful only to the application; since stream information is

> carried inside QUIC's encryption boundary, a given packet exposes no information about which
> stream(s) are carried within the packet.
> Therefore, stream multiplexing is not intended to be used for differentiating

Reviewer: ```suggestion
carried inside QUIC's encryption boundary, a given packet exposes
no information about which
stream(s) are carried within the packet.
Therefore, stream multiplexing is not intended to be used for differentiating
```

Streams are meaningful only to the application; since stream information is

> carried inside QUIC's encryption boundary, no information about the stream(s)
> whose frames are carried by a given packet is visible to the network.

Reviewer 2: I found this sentence a bit difficult to read, so a suggestion.

```suggestion
carried inside QUIC's encryption boundary, a given packet exposes no information about which
stream(s) are carried within the packet.
```
